CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# dbt_zendesk v0.24.1
+
+This release includes the following updates:
```

Comment:
This probably isn't necessary since the individual PRs are linked below.

Reply:
Whoever is the release reviewer - feel free to chime in here